docs/run/run-jobs-batch.mdx

```diff
@@ -8,11 +8,11 @@ description: How to run quantum computing jobs in batch mode using Qiskit Runtim
 
 Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
 
--  The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially. 
--  There is no delay between jobs, which can help avoid drift.
+-  The jobs' classical computation, such as compilation, is run in parallel (up to five jobs can be run simultaneously). Thus, running multiple jobs in a batch is significantly faster than running them serially.  
+-  There is minimal delay between jobs, which can help avoid drift. 
 
```

Comment:
After thinking about this some more, we can also add

- If you split your workload into multiple jobs and run them in Batch mode, you can get results from individual jobs. You can, for example, decide to cancel the rest of the jobs if the earlier job results don't meet your expectations. If one of the jobs fail, you can also re-submit just that one instead of re-running the entire workload.

Reply:
Actually, I'm not sure if we should mention job splitting now, or as part of the best practices. I don't want people to run tiny jobs after seeing this statement since there is an overhead associated with it.

Reply:
So you're thinking we should put it in the Sessions work instead ? https://github.com/Qiskit/documentation/pull/783 @jyu00

Reply:
Yeah let's just wait for that.